Libraries/libcar/Headers/car/Facet.h

```diff
@@ -59,6 +59,14 @@ class Facet {
 
 public:
     /*
+     * Load an existing rendition matching the provided attributes.
+     */
+    static Facet Load(
```

Comment:
Would this commit be better as a separate PR? It doesn't look too related to writing. (But maybe you use it later, haven't gotten there yet.)